Exercise_9/src/components/CustomLink.js

```diff
@@ -0,0 +1,15 @@
+import componentsService from './services';
+
+const customlinkComponent = {
+  template: '<a></a>',
+  beforeMount: (app, element, componentData) => {
+    componentsService.setAttributes(element, componentData);
+    const linkUrl = componentData.getNamedItem('href').value;
+    element.removeAttribute('href');
```

Comment:
You don't have to remove the attribute. You can just return `false` on `click`.
http://jsfiddle.net/sa6y5n34/